yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
```

Comment:
We changed this as they are the `a b c` parameters of the cell and not the full translation vectors of the cell